contents/tutorials/nextjs-app-directory-analytics.md

```diff
@@ -0,0 +1,264 @@
+```
+
+This opens a new page showing we are running Next.js 13.
+
+![Next.js app](../images/tutorials/nextjs-app-directory-analytics/app.png)
+
+## Setting up PostHog on the client side
+
+First, we need a PostHog instance ([signup for free](https://app.posthog.com/signup)). From this instance, we need a project API key, which is in project settings, and an instance address (`http://app.posthog.com`, `http://eu.posthog.com`, or a custom address). Add both of these to a `.env.local` file in our base directory.
+
+```
+NEXT_PUBLIC_POSTHOG_KEY=<ph_project_api_key>
+NEXT_PUBLIC_POSTHOG_HOST=<ph_instance_address>
+```
+
+Using the Next.js 13 app directory requires us to initialize PostHog differently than with [the pages directory](/tutorials/nextjs-analytics). Specifically, the app directory server-side renders components by default, and the `posthog-js` library is a client-side library. To make these work together, create a `providers.js` file and set up the `PostHogProvider` with the `'use client'`  directive.
+
+```js
+// app/providers.js
+'use client'
+import posthog from 'posthog-js'
+import { PostHogProvider } from 'posthog-js/react'
+
+if (typeof window !== 'undefined') {
+  posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY, {
+    api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST
+  })
+}
+
+export default function PHProvider({ children }) {
+  return <PostHogProvider client={posthog}>{children}</PostHogProvider>
+}
+```
+
+Once we do this, we can then import the `provider.js` file in our `app/layout.js` file, and wrap our app in the PostHog provider. 
+
+```js
+// app/layout.js
+import './globals.css'
+import Providers from './providers'
+
+export const metadata = {
+  title: 'Create Next App',
+  description: 'Generated by create next app',
+}
+
+export default function RootLayout({ children }) {
+  return (
+    <html lang="en">
+      <Providers>
+        <body>{children}</body>
+      </Providers>
+    </html>
+  )
+}
+```
+
+After setting this up, events start being autocaptured into your PostHog instance. This also means you can use the PostHog on the client side in all your client-side rendered Next.js components (the ones with the `"use client"` directive).
+
+![Events](../images/tutorials/nextjs-app-directory-analytics/events.png)
+
+> **Note:** if you don’t use the `"use client"` directive, Next.js assumes your page is server-side rendered. This means the client hooks like `usePostHog` cause errors. To interact with PostHog on the server side, use the [PostHog Node SDK](/docs/libraries/node) (which we show [later](#using-posthog-with-server-rendered-components)).
+
+## Capturing pageviews on the client side
+
+Next.js 13 with the app directory replaces `next/router` with `next/navigation.` This means we need a new implementation for capturing pageviews beyond the first-page load (because Next.js acts as a [single-page app](/tutorials/single-page-app-pageviews)). Luckily, we can add the code to handle this into our `provider.js` file.
```

Comment:
```suggestion
When used with the app directory, Next.js 13 replaces `next/router` with `next/navigation.` This means we need a new implementation for capturing pageviews beyond the first-page load because Next.js acts as a [single-page app](/tutorials/single-page-app-pageviews). Luckily, we can add the code to handle this into our `provider.js` file.
```